beanstalk_worker/worker_start_build.py

```diff
@@ -33,28 +32,30 @@
 DEBUG = 1
 DELAY = 30
 
+
 def debug_log(msg):
     if DEBUG == 1:
         logger.log(level=logging.DEBUG, msg=msg)
 
 
 def run_command(command):
-    debug_log("Running command: "+command)
+    debug_log("Running command: " + command)
     try:
         p = Popen(command, bufsize=0, shell=True,
-              stdout=PIPE, stderr=PIPE, stdin=PIPE)
-        #p.wait()
-        out,err = p.communicate()
+                  stdout=PIPE, stderr=PIPE, stdin=PIPE)
+        # p.wait()
+        out, err = p.communicate()
         return out
     except Exception as e:
         logger.log(level=logging.CRITICAL, msg=e.message)
         return e.message
 
+
 def notify_build_failure(namespace, notify_email, build_logs_file):
     msg_details = {}
     msg_details['action'] = 'notify_user'
     msg_details['namespace'] = namespace
-    msg_details['build_failed'] = True
+    msg_details['build_status'] = False
```

Comment:
How we are handling the failed build messages then?
https://github.com/CentOS/container-pipeline-service/blob/master/mail_service/worker_notify_user.py#L153